client-js/src/client/backend-client.js

```diff
@@ -268,23 +269,49 @@ export class BackendClient {
   /**
    * Sends the provided command to the server.
    *
+   * After sending the command to the server the following scenarios are possible:
+   * <ul>
+   *     <li>the {@param acknowledgedCallback} is called if the command is acknowledged for further processing
+   *     <li>the {@param errorCallback} is called if sending of the command failed
+   * </ul>
+   *
+   * Invocation of {@param acknowledgedCallback} and {@param errorCallback} are mutually exclusive.
```

Comment:
`@param` tag cannot be used for this according to both [Google code style](https://google.github.io/styleguide/jsguide.html#jsdoc-tags) and [JSDoc](http://usejsdoc.org/tags-param.html). Just mark the parameter names as code.

Use backticks (\`) instead of `{@code ... }` tag as [JSDoc supports Markdown](https://google.github.io/styleguide/jsguide.html#jsdoc-markdown). The HTML lists should also be replaced with Markdown alternatives.

Also, missing the articles:
```suggestion
   * Invocation of the {@param acknowledgedCallback} and the {@param errorCallback} are mutually exclusive.
```